doc/module-docs/cc_ubuntu_autoinstall/data.yaml

```diff
@@ -1,18 +1,21 @@
 cc_ubuntu_autoinstall:
   description: |
-    The ``autoinstall`` key indicates a configuration for the Ubuntu installer.
-    It is not acted on by cloud-init other than to ensure that the
-    configuration is schema compliant and that the installer package is
-    present on the system.
+    Cloud-init is used by the Ubuntu installer in two stages.
+    The ``autoinstall`` key may contain a configuration for the Ubuntu
+    installer.
+
+    Cloud-init verifies that an ``autoinstall`` key contains a ``version`` key
+    and that the installer package is present on the system.
 
     .. note::
-      The installer may use the provided configuration to instrument
-      cloud-init on the target system. See
-      `the Ubuntu installer documentation <https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#user-data>`_
-      for more information.
+      The autoinstaller might pass part of this configuration to cloud-init
```

Comment:
We're using "Ubuntu installer" and "autoinstaller" interchangeably here - could we standardise on one for consistency?